### PR TITLE
RUN-5538 - Expose findInPage and stopFindInPage APIs

### PR DESCRIPTION
**Pull request by luiemilio**
#### Description of Change

Expose Electron's Find In Page API

Core PR: https://github.com/HadoukenIO/core/pull/987

#### Checklist

- [X] PR description included and stakeholders cc'd
- [X] `npm test` passes
- [x] automated tests are [changed or added](https://testing-dashboard.openfin.co/#/app/dashboard)
- [X] Integration tests added in this repo and in the [test dashboard](https://testing-dashboard.openfin.co/#/app/dashboard)
- [ ] relevant documentation is changed or added
- [X] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)
- [X] PR release notes describe the change in a way relevant to app-developers
- [ ] PR has assigned reviewers

#### Release Notes

Added API to find and highlight text on a page.

Notes:

### CODE REVIEW

**Review by rdepena — CHANGES_REQUESTED** [review]
Needs proper docs and tests.

File: tutorials/Window.findInPage.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Find and highlight text on a page
```
Comment (rdepena) [inline tutorials/Window.findInPage.md]: This tutorial is not at all descriptive of what the feature does.

File: tutorials/Window.stopFindInPage.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Stop findInPage call with the specified action
```
Comment (rdepena) [inline tutorials/Window.stopFindInPage.md]: This tutorial is not at all descriptive of what the feature does.

**Review by rdepena — CHANGES_REQUESTED** [review]

File: src/api/webcontents/webcontents.ts [hunk 3]
```diff
@@ -41,4 +41,13 @@ export class WebContents<T extends WebContentsEventMapping> extends EmitterBase<
             ignoreCache
         }, this.identity)).then(() => undefined);
     }
-}
\ No newline at end of file
+
+    public findInPage(searchTerm: string, options?: any): Promise<number> {
```
Comment (licui3936) [inline src/api/webcontents/webcontents.ts]: The type of 'options' is FindInPageOptions ?

File: tutorials/Window.findInPage.md (hunk 1 above)
Comment (luiemilio) [reply tutorials/Window.findInPage.md]: added better description

File: tutorials/Window.stopFindInPage.md (hunk 2 above)
Comment (luiemilio) [reply tutorials/Window.stopFindInPage.md]: added better description

File: src/api/webcontents/webcontents.ts (hunk 3 above)
Comment (luiemilio) [reply src/api/webcontents/webcontents.ts]: added the type to the func

**Review by licui3936 — APPROVED** [review]

**Pull request closed without merging**